packages/firestore/src/local/simple_db.ts

```diff
@@ -312,7 +314,8 @@ export class SimpleDbTransaction {
     this.transaction.oncomplete = () => {
       this.completionDeferred.resolve();
     };
-    this.transaction.onabort = () => {
+    this.transaction.onabort = (event) => {
```

Comment:
Is this correct? `event` is not getting used.

Reply:
Oh, that could be reverted...  I was just hacking this together real quick and am not planning to submit it as-is. 😄

Reply:
Alright, just wanted to be sure where you'd want the error from. 😅 

I'm pushing a new release with your suggestions here; so we should have some kind of data fairly soon.

Reply:
Ah, fantastic!  I think while writing this I was wondering the same thing (if there was error information in the `event`) but discovered that there was not.